main.py

```diff
@@ -0,0 +1,10 @@
+from typing import Union
+
+def add(x: Union[int, float], y: Union[int, float]) -> Union[int, float]:
+    return x + y
```

Comment:
예외처리가 누락되어있어요. 다음과 같이 수정하는 것은 어떨까요?

```suggestion
from typing import Union, Optional

def add(x: Union[int, float], y: Union[int, float]) -> Optional[Union[int, float]]:
    try:
        result = x + y
    except Exception as e:
        print(e)
        return None
    return result
```

Reply:
에러 발생 시 None으로 반환하지말고 TypeError를 반환하는 것으로 처리하겠습니다.